examples/annotations/main.css

```diff
@@ -4,7 +4,7 @@
   position: absolute;
   left: 10px;
   top: 80px;
-  z-index: 1;
+  z-index: 1000;
```

Comment:
When layers are children of other layers, we set a z-index on them.  When we have multiple layers, this conflicts with the z-index context of the controls.  The easy solution is to move the controls to an arbitrary high z-index.  It might be better to ensure that the map is a stacking context, which on some browsers would be adding css `isolation: isolate`, but that doesn't work everywhere and other methods of creating a stacking context involve altering css properties on the map div which might be undesirable in general (we could do so in the examples, though).